src/async-wrap.cc

```diff
@@ -164,8 +168,6 @@ static void DestroyIdsCb(uv_idle_t* handle) {
       FatalException(env->isolate(), try_catch);
     }
   }
-
-  env->destroy_ids_list()->clear();
```

Comment:
This would be nice for the purpose of adding a good commit message descibing the issue.